stl/inc/xutility

```diff
@@ -4688,15 +4724,16 @@ _INLINE_VAR constexpr bool _Equal_memcmp_is_safe_helper<_Elem1, _Elem2,
                     && !is_volatile_v<_Elem1> && !is_volatile_v<_Elem2>,
         equal_to<>>> = _Value_equality_is_bitwise_equality<_Elem1, _Elem2>;
 
-// return equal optimization category for pointers
-template <class _Elem1, class _Elem2, class _Pr, bool = conjunction_v<is_pointer<_Elem1>, is_pointer<_Elem2>>>
+// Determine if it's safe to compare pointers

```

Comment:
We aren't going to compare _pointers_ we're going to compare _pointees_ - the things the pointers point at.